contracts/modules/common/IModule.sol

```diff
@@ -24,39 +24,9 @@ pragma solidity >=0.5.4 <0.7.0;
  * @author Julien Niset - <julien@argent.xyz>
  */
 interface IModule {
-
-    enum OwnerSignature {
-        Anyone,             // Anyone
-        Required,           // Owner required
-        Optional,           // Owner and/or guardians
-        Disallowed          // guardians only
-    }
-
-    /**
-    * @notice Utility method to recover any ERC20 token that was sent to the module by mistake.
-    * @param _token The token to recover.
-    */
-    function recoverToken(address _token) external;
-
     /**
      * @notice Inits a module for a wallet by e.g. setting some wallet specific parameters in storage.
      * @param _wallet The wallet.
      */
     function init(address _wallet) external;
-
-    /**
-     * @notice Adds a module to a wallet. Cannot execute when wallet is locked (or under recovery)
-     * @param _wallet The target wallet.
-     * @param _module The modules to authorise.
-     */
-    function addModule(address _wallet, address _module) external;
```

Comment:
I think we should keep that method as is it critical to have it if we want to update the `VersionManager`.